### PR TITLE
Update ALVR.ALVR to 20.4.3

### PR DESCRIPTION
**Pull request by TobiX**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124376)

### CODE REVIEW

**Comment by TobiX** [comment]
> The package manager bot determined there was an issue with installing the application correctly.

It seems SmartScreen doesn't like the file 😞

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978334957 
2023-10-31 11:41:00.540 [CLI ] ShellExecute installer failed: 1633 
MSI (c) (84:AC) [11:41:00:066]: Product: ALVR -- Installation failed. 
MSI (c) (84:AC) [11:41:00:066]: Windows Installer installed the product. Product Name: ALVR. Product Version: 20.4.3. Product Language: 1033. Manufacturer: alvr-org. Installation success or error status: 1633. 
[1E84:15D8][2023-10-31T11:40:58]w363: Could not create system restore point, error: 0x80070422. Continuing... 
[1E84:15D8][2023-10-31T11:41:00]e000: Error 0x80070661: Failed to install MSI package. 
[1E84:15D8][2023-10-31T11:41:00]e000: Error 0x80070661: Failed to execute MSI package. 
[1EA4:1EE8][2023-10-31T11:41:00]e000: Error 0x80070661: Failed to configure per-machine MSI package. 
[1EA4:1EE8][2023-10-31T11:41:00]e000: Error 0x80070661: Failed to execute MSI package.

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/a/ALVR/ALVR/20.4.3/ALVR.ALVR.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageIdentifier: ALVR.ALVR
+PackageVersion: 20.4.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: burn
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{758d1530-122b-4d0f-8ae2-f02938ff6c21}'
```
Comment (stephengillie) [inline manifests/a/ALVR/ALVR/20.4.3/ALVR.ALVR.installer.yaml:13]:
```suggestion
ProductCode: '{758d1530-122b-4d0f-8ae2-f02938ff6c21}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Pull request closed without merging**